README.md

```diff
@@ -68,6 +68,17 @@ echo "POSTGRES_PASSWORD=$(tr -dc A-Za-z0-9 </dev/urandom | head -c 32)" >> .env
 
 It is recommended to let the other environment variables as they are
 
+---
```

Comment:
I think adding indications of the folder in witch wee need to run the commands would be a good idea. As they do not have to be run in the root folder